Add Sample Documentation (#55)
LM-Development/aks-sample: Samples/PublicSamples/RecordingBot/docs/explanations/recording-bot-permission.md

| [Calls.JoinGroupCall.All](https://learn.microsoft.com/en-us/graph/permissions-reference#callsjoingroupcallall) | Allows the app to join group calls and scheduled meetings in your organization, without a signed-in user. The app will be joined with the privileges of a directory user to meetings in your tenant. |
| [Calls.JoinGroupCallasGuest.All](https://learn.microsoft.com/en-us/graph/permissions-reference#callsjoingroupcallasguestall) | Allows the app to anonymously join group calls and scheduled meetings in your organization, without a signed-in user. The app will be joined as a guest to meetings in your tenant. |
| [Calls.AccessMedia.All](https://learn.microsoft.com/en-us/graph/permissions-reference#callsaccessmediaall) | Allows the app to get direct access to participant media streams in a call, without a signed-in user. |

Not all of them are necessary for a compliance recording bots. A compliance recording only requires:

- Calls.AccessMedia.All
- Calls.JoinGroupCall.All
- Calls.JoinGroupCallAsGuest.All

> [!IMPORTANT]  

> > After configuring the application permissions it is required that a Microsoft Entra Id adminstrator grants the permissions, this also applies for any time the application permissions are changed. Changes made to the application permissions of an app registration will not reflect until a Microsoft Entra Id administarator has consented to them.

Reviewer: ```suggestion
> After configuring the application permissions it is required that a Microsoft Entra Id administrator grants the permissions. This also applies for any time the application permissions are changed. Changes made to the application permissions of an app registration will not take effect until a Microsoft Entra Id administarator has consented to them.

```

| [Calls.JoinGroupCall.All](https://learn.microsoft.com/en-us/graph/permissions-reference#callsjoingroupcallall) | Allows the app to join group calls and scheduled meetings in your organization, without a signed-in user. The app will be joined with the privileges of a directory user to meetings in your tenant. |
| [Calls.JoinGroupCallasGuest.All](https://learn.microsoft.com/en-us/graph/permissions-reference#callsjoingroupcallasguestall) | Allows the app to anonymously join group calls and scheduled meetings in your organization, without a signed-in user. The app will be joined as a guest to meetings in your tenant. |
| [Calls.AccessMedia.All](https://learn.microsoft.com/en-us/graph/permissions-reference#callsaccessmediaall) | Allows the app to get direct access to participant media streams in a call, without a signed-in user. |

But not all of them are necessary for a compliance recording bots. For a compliance recording bot are only

- Calls.AccessMedia.All
- Calls.JoinGroupCall.All
- Calls.JoinGroupCallAsGuest.All

> permissions required.

Reviewer 2: ```suggestion

```

> [!IMPORTANT]  

> > After configuring the application permissions it is required that a Microsoft Entra Id adminstrator grants the permission, this also applys for any time the application permissions are changed, changes made to the application permissions of an app registration will not reflect until consent of a Microsoft Entra Id administarator has been reapplied.

Reviewer 2: ```suggestion
> After configuring the application permissions it is required that a Microsoft Entra Id adminstrator grants the permissions, this also applies for any time the application permissions are changed. Changes made to the application permissions of an app registration will not reflect until a Microsoft Entra Id administarator has consented to them.

```

> It is possible for administrators to grant the application permissions in the [Azure portal](https://portal.azure.com), but often a better option is to provide a sign-up experience for administrators by using the Microsoft Entra Id `/adminconsent` endpoint, to do that see also the [instructions on constructing an Admin Consent URL](https://learn.microsoft.com/en-us/entra/identity-platform/v2-admin-consent).

Reviewer 2: ```suggestion
It is possible for administrators to grant the application permissions in the [Azure portal](https://portal.azure.com), but often a better option is to provide a sign-up experience for administrators by using the Microsoft Entra Id `/adminconsent` endpoint.  
see [instructions on constructing an Admin Consent URL](https://learn.microsoft.com/en-us/entra/identity-platform/v2-admin-consent).

```
